CHANGELOG.rst

```diff
@@ -1,9 +1,10 @@
 Changes
 =======
 
-0.20.0 (2022-08-23)
+0.20.0 (2022-8-22)
 -------------------
 
+* Moving TLS certificate management to another library called``django-simple-certmanager``. Managing Certificates with this library remains possible for now, but please consider to update your project.
```

Comment:
you'll need a separate changelog entry for this - 0.20.0 has been released to PyPI on August 23rd already.